### PR TITLE
Support turbolinks 5 and previous versions

### PR DESCRIPTION
**Pull request by awead**

### CODE REVIEW

File: app/assets/javascripts/browse_everything/behavior.js.coffee [hunk 1]
```diff
@@ -304,7 +304,11 @@ auto_toggle = ->
   triggers = $('*[data-toggle=browse-everything]')
   triggers.each () -> $(this).browseEverything($(this).data())
 
-if Turbolinks?
-  $(document).on 'page:change', auto_toggle
+if Turbolinks && Turbolinks.supported
```
Comment (cbeer) [inline app/assets/javascripts/browse_everything/behavior.js.coffee]: How does `Turbolinks?` and `Turbolinks` compile differently?

Comment (awead) [inline app/assets/javascripts/browse_everything/behavior.js.coffee]: Don't know. I was just copying what you did in Blacklight.

Comment (cbeer) [inline app/assets/javascripts/browse_everything/behavior.js.coffee]: Here's what we have in Blacklight:

```
if (typeof Turbolinks !== 'undefined' && Turbolinks.supported) {
```

I'm just not sure if `if Turbolinks` compiles down to that?

Comment (awead) [inline app/assets/javascripts/browse_everything/behavior.js.coffee]: Ah, ok. I was looking at an earlier commit. I'll update this.

**Pull request merged**